src/test/java/seedu/address/model/RestaurantBookTest.java

```diff
@@ -114,22 +116,29 @@ public void removeListener_withInvalidationListener_listenerRemoved() {
      */
     private static class RestaurantBookStub implements ReadOnlyRestaurantBook {
         private final ObservableList<Member> members = FXCollections.observableArrayList();
-        private final ObservableList<Booking> bookings = FXCollections.observableArrayList();
```

Comment:
why is this deleted?

Reply:
As you can see in this test, only member-level operations are tested. In the event that we need this method, we can always put it back.

As of now, this list does not serve any purpose.

Reply:
ok, but later should test all 4